src/App.js

```diff
@@ -1,23 +1,74 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import HomePage from "./components/HomePage";
+import Beers from "./components/Beers";
+import BeerDetails from "./components/BeerDetails";
+import CreateBeer from "./components/CreateBeer";
+import RandomBeer from "./components/RandomBeer";
+import { Routes, Route } from "react-router-dom";
+import axios from "axios";
+import { useState, useEffect } from "react";
 
 function App() {
+  const [beers, setBeers] = useState(null);
+
+  useEffect(() => {
+    fetchBeers();
+  }, []);
+
+  const baseURL = "https://ih-beers-api2.herokuapp.com";
+  // .get(process.env.REACT_APP_API_BASE_URL + "/beers")
+
+  const fetchBeers = () => {
+    axios
+      .get(baseURL + "/beers")
+      .then((response) => {
+        const beers = response.data;
+        setBeers(beers);
+      })
+      .catch((e) => {
+        console.log(e);
+      });
+  };
+
+  const createBeer = (beerInfo) => {
+    axios
+      .post(baseURL + "/beers/new", beerInfo)
+      .then((response) => {
+        console.log(response.data);
+      })
+      .catch((e) => {
+        console.log(e);
+      });
+  };
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <HomePage />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
```

Comment:
You could remove that `Route` in that case you will get a home page with only one group of links